Guardfile

```diff
@@ -38,31 +38,48 @@ group :red_green_refactor, halt_on_fail: true do
+    watch('app/controllers/v2/reservations_controller.rb') { 'spec/features/person_responds_to_interview_invitation_over_email_spec.rb' }
+    watch('app/controllers/public/people_controller.rb') { 'spec/features/people_registration_spec.rb' }
+
     watch('app/views/v2/event_invitations/new.html.erb') { 'spec/features/invite_person_to_phone_call_spec.rb' }
+    watch('app/views/v2/event_invitations/new.html.erb') { 'spec/features/sms_invitation_to_phone_call_spec.rb' }
     watch('app/views/v2/event_invitations/_form.html.erb') { 'spec/features/invite_person_to_phone_call_spec.rb' }
+    watch('app/views/v2/event_invitations/_form.html.erb') { 'spec/features/sms_invitation_to_phone_call_spec.rb' }
+    watch('app/views/v2/reservations/new.html.erb') { 'spec/features/person_responds_to_interview_invitation_over_email_spec.rb' }
+    watch('app/views/v2/reservations/_form.html.erb') { 'spec/features/person_responds_to_interview_invitation_over_email_spec.rb' }
     watch('app/views/event_invitation_mailer/invite.html.erb') { 'spec/features/invite_person_to_phone_call_spec.rb' }
+    watch('app/views/event_invitation_mailer/invite.html.erb') { 'spec/features/person_responds_to_interview_invitation_over_email_spec.rb' }
+    watch('app/views/event_invitation_mailer/invite.html.erb') { 'spec/features/sms_invitation_to_phone_call_spec.rb' }
     watch('app/views/event_invitation_mailer/invite.text.erb') { 'spec/features/invite_person_to_phone_call_spec.rb' }
-
-    watch('spec/features/person_responds_to_interview_invitation_spec.rb')
-    watch('app/views/event_invitation_mailer/invite.html.erb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-    watch('app/views/event_invitation_mailer/invite.text.erb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-
-    watch('app/mailers/event_invitation_mailer.rb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-    watch('app/models/v2/reservation.rb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-    watch('app/controllers/v2/reservations_controller.rb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-    watch('app/views/v2/reservations/new.html.erb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
-    watch('app/views/v2/reservations/_form.html.erb') { 'spec/features/person_responds_to_interview_invitation_spec.rb' }
+    watch('app/views/event_invitation_mailer/invite.text.erb') { 'spec/features/person_responds_to_interview_invitation_over_email_spec.rb' }
+    watch('app/views/event_invitation_mailer/invite.text.erb') { 'spec/features/sms_invitation_to_phone_call_spec.rb' }
+    watch('app/views/public/people/new.html.erb') { 'spec/features/people_registration_spec.rb' }
+    watch('app/views/public/people/_form.html.erb') { 'spec/features/people_registration_spec.rb' }
   end
```

Comment:
Doesn't seem to be a sustainable approach to list so many individual files with their individual dependencies here. I'd rather only list patterns and run a few tests that might not be necessary to run each time than having to keep this list up to date.